### PR TITLE
fix: bot response

### PR DESCRIPTION
**Pull request by trkhoi**
- Requirements: https://www.notion.so/console-labs/Check-all-bot-response-a843ba87c3fe406c9b7bf03b388c64b8
image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: src/commands/community/stats.ts [hunk 1]
```diff
@@ -47,7 +47,9 @@ const countStatsHandler: CommandChoiceHandler = async (msgOrInteraction) => {
   await Community.createStatChannel(message.guildId, countTypeReq)
   const successEmbeded = composeEmbedMessage(message, {
     title: `Server Stats\n\n`,
-    description: `Successfully count ` + type + ` ` + stat,
+    description: `${
+      type.charAt(0).toUpperCase() + type.slice(1)
```
Comment (anhnh12) [inline src/commands/community/stats.ts:51]: use capFirst()

File: src/commands/community/verify_slash/set.ts [hunk 2]
```diff
@@ -41,7 +41,7 @@ export async function verifySet(interaction: CommandInteraction) {
       messageOptions: {
         embeds: [
           getErrorEmbed({
-            description: "Invalid channel",
+            description: "0x7aCeE5D0acC520faB33b3Ea25D4FEEF1FfebDE73 250",
```
Comment (anhnh12) [inline src/commands/community/verify_slash/set.ts:44]: ???

File: src/commands/config/levelRole/remove.ts [hunk 3]
```diff
@@ -42,7 +42,9 @@ const command: Command = {
         messageOptions: {
           embeds: [
             composeEmbedMessage(msg, {
-              description: `Level-role configuration removed for lv${level}`,
+              description:
+                `Level-role configuration removed for lv${level}.` +
+                " To set a new one, run `$lr <role> <level>`.",
```
Comment (anhnh12) [inline src/commands/config/levelRole/remove.ts:47]:
```suggestion
              description:
                `Level-role configuration removed for lv${level}.\nTo set a new one, run \`$lr <role> <level>\`.`,
```

**Review by anhnh12 — APPROVED** [review]

File: src/commands/community/nft/add.ts [hunk 4]
```diff
@@ -83,8 +83,8 @@ export async function toEmbed(
       ) {
         return buildDiscordMessage(
           msg,
-          "NFT",
-          "Already added. Nft is in sync progress"
+          "Existing Collection",
+          "Please add another one or view the collection by  `$nft <collection_symbol> <token_id>`."
```
Comment (tuanddd) [inline src/commands/community/nft/add.ts:87]:
```suggestion
          "Please add another one or view the collection by `$nft <collection_symbol> <token_id>`."
```

File: src/commands/config/levelRole/remove.ts [hunk 5]
```diff
@@ -42,7 +42,7 @@ const command: Command = {
         messageOptions: {
           embeds: [
             composeEmbedMessage(msg, {
-              description: `Level-role configuration removed for lv${level}`,
+              description: `Level-role configuration removed for lv${level}.\nTo set a new one, run \`$lr <role> <level>\`.`,
```
Comment (tuanddd) [inline src/commands/config/levelRole/remove.ts:45]: suggestion: bỏ dấu gạch ngang

**Review by tuanddd — CHANGES_REQUESTED** [review]

**Review by tuanddd — APPROVED** [review]

**Pull request merged**